docs/metadata/prophecy-build-tool.md

```diff
@@ -17,9 +17,9 @@ tags:
 PySpark Pipelines) to integrate with your own CI / CD (e.g. Github Actions), build system (e.g. Jenkins), and
 orchestration (e.g. Databricks Workflows).
 
-## Features (v1.0.4.1)
+## Features (v1.0.5)
 
-- Build and unit test all Pipelines in Prophecy projects (Scala and Python)
+- Build (with Pipeline filters) and unit test all Pipelines in Prophecy projects (Scala and Python)
```

Comment:
Build and unit test specified Scala and Python Pipelines in a Prophecy Project. (Build all pipelines or specify which ones to build)